### PR TITLE
Feature/cat list

### PR DESCRIPTION
**Pull request by Ajaybadhe23**

### CODE REVIEW

File: src/screen/addCat.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                                            <span className='text-rose-600 dark:text-rose-500 text-sm'>{NameErrorMsg}</span>
+                                        </div>
+
+                                        <div className="col-span-12 sm:col-span-3">
+                                            <label htmlFor="country" className="block text-sm font-medium text-gray-700">Breed <sup>*</sup></label>
+                                            <select id="country" value={breed} onChange={catBreedHandler} name="country" autoComplete="country-name" className='mt-1 block w-full py-2 px-3 border border-gray-300 bg-white rounded-md shadow-sm  sm:text-sm focus:border focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 '>
```
Comment (GSKulkarni9860) [inline src/screen/addCat.js:111]: Can we use dynamic list here for options?

File: src/store/reducer/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from "redux";
+import { catListReducer } from "./listReducer";
+
+const rootReducers = combineReducers(
```
Comment (GSKulkarni9860) [inline src/store/reducer/index.js:4]: no need to use combineReducers() when we have only one reducer

File: src/store/reducer/listReducer.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { GET_LIST_SUCCESS } from "../constant/constant"
+
+let lists = JSON.parse(localStorage.getItem('list'))
+const initialState = {
+    catList: lists || []
```
Comment (GSKulkarni9860) [inline src/store/reducer/listReducer.js:5]: set data from component. create a separate action for setting local stored data

**Review by GSKulkarni9860 — CHANGES_REQUESTED** [review]

File: src/screen/addCat.js [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                catName,
+                description,
+                breed,
+                id: (new Date().getTime()).toString(36)
+            }
+            setData([catData, ...data])
+            getCatListAction(dispatch, [catData, ...data])
+        }
+        else {
+            let updatedData = data.map((curElem) => {
```
[the remark this answers is not in the document]
Comment (Ajaybadhe23) [reply src/screen/addCat.js:68]: not able to update using find

**Review by GSKulkarni9860 — APPROVED** [review]

**Pull request merged**